resources/css/app.css

```
@@ -560,3 +560,77 @@
 }
 .prose-ticket th { background: #FAFAF8; font-weight: 600; }
 .prose-ticket hr { border: 0; border-top: 1px solid #E2E0D8; margin: 1rem 0; }
+
+/* ProseMirror base */
+.ProseMirror {
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)